src/chunk/chunk.rs

```diff
@@ -86,5 +86,7 @@ pub fn chunks(ch: Chunk) -> Vec<Chunk> {
 pub enum TupleMode {
     Auto, // "", "1", "(1,2)"
     WithUnit, // "()", "1", "(1,2)"
+    #[deprecated]
+    #[allow(dead_code)]
     Simple,    // "()", "(1)", "(1,2)"
```

Comment:
Why don't you remove it yet? Is it still in use elsewhere?

Reply:
It was commented before @antoyo decommented it when adding asyncs,
so I decided on my whim that better not comment it but deprecate 😉